src/main/java/net/geforcemods/securitycraft/items/UniversalOwnerChangerItem.java

```diff
@@ -113,4 +116,49 @@ else if(BlockUtils.getBlock(world, pos.down()) instanceof ReinforcedDoorBlock ||
 
 		return ActionResultType.FAIL;
 	}
+
+	@Override
+	public ActionResult<ItemStack> onItemRightClick(World world, PlayerEntity player, Hand hand) {
+		ItemStack ownerChanger = player.getHeldItem(hand);
+
+		if (!world.isRemote) {
+			if (!ownerChanger.hasDisplayName()) {
+				PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize(SCContent.UNIVERSAL_OWNER_CHANGER.get().getTranslationKey()), ClientUtils.localize("messages.securitycraft:universalOwnerChanger.noName"), TextFormatting.RED);
+				return ActionResult.resultFail(ownerChanger);
+			}
+
+			if (hand == Hand.MAIN_HAND && player.getHeldItemOffhand().getItem() == SCContent.BRIEFCASE.get()) {
+				ItemStack briefcase = player.getHeldItemOffhand();
+
+				if (isOwnedBy(player, briefcase)) {
+					String newOwner = ownerChanger.getDisplayName().getString();
+
+					if (!briefcase.hasTag())
+						briefcase.setTag(new CompoundNBT());
+
+					briefcase.getTag().putString("owner", newOwner);
+					briefcase.getTag().putString("ownerUUID", PlayerUtils.isPlayerOnline(newOwner) ? PlayerUtils.getPlayerFromName(newOwner).getName().toString() : "ownerUUID");
+					PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize(SCContent.UNIVERSAL_OWNER_CHANGER.get().getTranslationKey()), ClientUtils.localize("messages.securitycraft:universalOwnerChanger.changed", newOwner), TextFormatting.GREEN);
+
+					return ActionResult.resultSuccess(ownerChanger);
+				}
+				else {
+					PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize(SCContent.UNIVERSAL_OWNER_CHANGER.get().getTranslationKey()), ClientUtils.localize("messages.securitycraft:universalOwnerChanger.briefcase.notOwned"), TextFormatting.RED);
+				}
+			}
+		}
+
+		return ActionResult.resultFail(ownerChanger);
+	}
+
+	private boolean isOwnedBy(PlayerEntity player, ItemStack briefcase) {
+		if (!briefcase.hasTag())
+			return true;
+		else if (!briefcase.getTag().contains("owner"))
+			return true;
+		else if (briefcase.getTag().getString("ownerUUID").equals(player.getUniqueID().toString()) || (briefcase.getTag().getString("ownerUUID").equals("ownerUUID") && briefcase.getTag().getString("owner").equals(player.getName().getString())))
+			return true;
+		else
+			return false;
+	}
```

Comment:
With the proposed change above, this can be removed and the other method called statically.